themes/openy_themes/openy_rose/templates/paragraph/paragraph--banner.html.twig

```diff
@@ -62,9 +62,11 @@
           <div class="banner-description">
             {{ content.field_prgf_description}}
           </div>
-          <div class="btn banner-btn ">
-            {{ content.field_prgf_link }}
-          </div>
+          {% if paragraph.field_prgf_link.0.uri %}
```

Comment:
It is safe to use content here, not a paragraph
If uri is not defined - you'll get error 500 on template level